--- zlibrary/core/src/unix/library/ZLibrary.cpp 2012-10-21 01:51:16.765894674 +0200
+++ zlibrary/core/src/unix/library/ZLibrary.cpp.new     2012-10-21 01:51:32.477791479 +0200
@@ -23,7 +23,7 @@
 #include <sys/stat.h>
 
 #include <algorithm>
-
+#include <zlib.h>
 #include <ZLibrary.h>
 #include <ZLStringUtil.h>
 #include <ZLLogger.h>

--- makefiles/arch/desktop.mk   2010-04-01 15:14:24.000000000 +0200
+++ makefiles/arch/desktop.mk.new       2012-10-21 11:11:08.000000000 +0200
@@ -17,7 +17,7 @@
   MOC = moc-qt3
   QTINCLUDE = -I /usr/include/qt3
 else
-  MOC = moc-qt4
+  MOC = moc
   QTINCLUDE = -I /usr/include/qt4
 endif
 
 
 